build/NuSpecs/MUXControls-Nuget-FrameworkPackage.props

```diff
@@ -117,14 +117,16 @@
   <!-- Workaround for https://github.com/Microsoft/microsoft-ui-xaml/issues/599. 
   If there are more than two projects are using microsoft-ui-xaml framework package, and the projects have reference relationship,
   more than two compact.xbf with the same destination path would be packaged and compiler would complains error. 
-  To make sure only one compact.xbf is packaged, this target excludes the compact.xbf which doesn't match MSBuildProjectName.
+  Assume we have two projects A and B, and A->B->microsoft-ui-xaml, compact.xbf is built into B.pri, and then built into A.pri because of project reference.
+  So we need to exclude compact.xbf in A. To make sure only one compact.xbf is packaged, this target excludes the compact.xbf which matches MSBuildProjectName.
   --> 
```

Comment:
Hmm ... so what happens if MUX is included by way of multiple projects like 
```
 A->B->MUX
 \->C->MUX
```

Will we still run into this because B and C both have the resource?

I guess I don't understand why we don't see this today where MUX prerelease nuget package includes a .pri file with resources that would be duplicates. Can you share out the binlog of the failure case?

Reply:
no. I did the testing and didn't find any problem. B.pri and C.pri includes compact.xbf. when compiling A. compact.xbf on B and C would be copied/overwritten project by project, so I didn't see any problem.
For prerelease, if I add compact page to the app with the same layout, I think it also have conflicts.
I will provide a binlog tomorrow. also I can verify the prerelease assumption.

Reply:
So the condition is still bothering me. I think the goal is "if there are multiple items whose path ends with 'Microsoft.UI.Xaml\DensityStyles\Compact.xbf' then we know that's ours and it doesn't matter which one is chosen, so remove all but one of them". But that's not exactly what this is doing. Can you figure out how to change the condition so it just selects one from the set?